tests/unit/plugins/modules/test_jenkins_build.py

```diff
@@ -12,6 +12,7 @@
 from ansible_collections.community.general.plugins.modules import jenkins_build
 
 import json
+import jenkins
```

Comment:
If you write this as
```suggestion
import pytest

jenkins = pytest.importorskip('jenkins')
```
then the test will still work when `jenkins` cannot be imported.

But: why not simply also mock JenkinsException, now that everything else is already mocked, and use that, instead of increasing the set of unit test requirements?

Reply:
The suggested change looks good.
However, looking at the error logs, I assume the CI error is not caused by my code patch.
 
___ERROR collecting gw0 ___
Different tests were collected between gw1 and gw0. The difference is:
01:01 --- gw1
01:01 
01:01 +++ gw0

01:01 -lookup/onepassword/test_onepassword.py::test_op_lookup[OnePassCLIv2-Test Vault-queries3-kwargs3-output3-expected3]
01:01 -lookup/onepassword/test_onepassword.py::test_op_lookup[OnePassCLIv2-Test Vault-queries4-kwargs4-output4-expected4]
01:01 -lookup/onepassword/test_onepassword.py::test_op_lookup[OnePassCLIv2-Test Vault-queries5-kwargs5-output5-expected5]
01:01 +lookup/onepassword/test_onepassword.py::test_op_lookup[OnePassCLIv2-Test Vault-queries0-kwargs0-output0-expected0]
01:01 +lookup/onepassword/test_onepassword.py::test_op_lookup[OnePassCLIv2-Test Vault-queries1-kwargs1-output1-expected1]
01:01 +lookup/onepassword/test_onepassword.py::test_op_lookup[OnePassCLIv2-Test Vault-queries2-kwargs2-output2-expected2]
01:01 +lookup/onepassword/test_onepassword.py::test_op_lookup[OnePassCLIv1-Acme "Quot'd" Servers-queries3-kwargs3-output3-expected3]
01:01 +lookup/onepassword/test_onepassword.py::test_op_lookup[OnePassCLIv1-Acme Logins-queries4-kwargs4-output4-expected4]
01:01 +lookup/onepassword/test_onepassword.py::test_op_lookup[OnePassCLIv1-Acme Logins-queries5-kwargs5-output5-expected5]
01:01  lookup/onepassword/test_onepassword.py::test_signin[opv1]
01:01  lookup/onepassword/test_onepassword.py::test_signin[opv2]

Reply:
The CI errors are unrelated, yes, and they're fixed by now.